pkg/JuliaInterface/src/convert.c

```diff
@@ -27,23 +28,28 @@ jl_value_t * julia_gap(Obj obj)
     if (obj == False) {
         return jl_false;
     }
+    jl_value_t * result = (jl_value_t *) obj;
```

Comment:
```suggestion
    jl_value_t * result = (jl_value_t *)obj;
```
otherwise fails the clang-format check

Reply:
Fixed. I had run clang-format over it for the previous commit, but forgot it this time.